pup-0003.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Cherry-Picking Commits
+author: David Davis
+created: 01-Apr-2017
+status:
+---
+
+## Summary
+
+Currently in Pulp, we merge features to master. However, hotfixes and bug fixes
+are merged to release branches which then are merged to newer release branches
+and then to master (a process called "merging forward"). More info about
+merging forward can be found in our documentation:
+
+http://docs.pulpproject.org/en/nightly/dev-guide/contributing/merging.html#merging-your-changes
+
+Also, a good illustration of this process is outlined in the following webpage:
+
+http://nvie.com/posts/a-successful-git-branching-model/
+
+This PUP is to use cherry-picking instead of merging forward. Using this model,
+all PRs would be made against master and then changes would be cherry-picked to
+release branches as needed.
```

Comment:
@goosemania you say <current>-release but could it also be an older release? Like if we found say a security issue in 2.9, we would cherry-pick the change to 2.9-release, right? Then would we cherry-pick the change to 2.10-release, and so forth?